src/4.3_to_5.0/_cleanupDescendants.test.ts

```diff
@@ -7,7 +7,7 @@ const cube = dataModelsForTests.sandbox.catalogs[0].cubes[0];
 
 describe("_cleanupDescendants", () => {
   it("replaces a Descendants function by its input set if the shallowest level expressed in the set is the same as the target one or above", () => {
-    const mdx = parse(`                                 
+    const mdx: MdxSelect = parse(`                                 
```

Comment:
It's necessary to avoid having a stactic type error.
In my dev I decided to enforce the input type of the  `_cleanupDescendants()`, from previously `Mdx` to `MdxSelect | MdxDrillthrought`, as the function is only used in `migrateQuery` which accepts only the two mdx types.
Still, I decided to add the type in the `parse` function. But let me know if I should change back the type variable of the `_cleanupDescendants()`